data/network_delegation/init.go

```diff
@@ -7,11 +7,17 @@ import (
 	"github.com/Oneledger/protocol/storage"
 )
 
-const DELEGATION_POOL_KEY = "00000000000000000001"
-
 var logger *log.Logger
 var prefixMap map[DelegationPrefixType]string
 
+const (
+	MatureType  DelegationPrefixType = 0x103
+	PendingType DelegationPrefixType = 0x102
+	ActiveType  DelegationPrefixType = 0x101
+
+	DELEGATION_POOL_KEY = "00000000000000000001"
```

Comment:
We can probably put the hardcoded values of the delegation options from here, and call this from application.do